VSTS build fails with error

VSTS shows error 

##[error]Invoke-WebRequest : The response content cannot be parsed because the Internet Explorer engine is not available, or 
Internet Explorer's first-launch configuration is not complete. Specify the UseBasicParsing parameter and try again.

--- a/build/CheckNuGetVersion.ps1
+++ b/build/CheckNuGetVersion.ps1
@@ -4,8 +4,8 @@ $matches = [regex]::Match($nuspec, "<version>(.*?)<\/version>")
 $localVersion = $matches[0].Captures.Groups[1].Value
 # get version on nuget.org
 $url = "https://api.nuget.org/v3-flatcontainer/devdeer.DoctorFlox/index.json"
-$versions = Invoke-WebRequest $url | ConvertFrom-Json | Select -expand versions
+$versions = Invoke-WebRequest $url -UseBasicParsing | ConvertFrom-Json | Select -expand versions
 $nugetVersion = $versions[$versions.Length - 1]
 # compare versions
-$result = $localVersion.Equals($nugetVersion)
+$result = !$localVersion.Equals($nugetVersion)
 Write-Host ("##vso[task.setvariable variable=NuGet.PushRequired;]$result")

